Source/WTF/wtf/SHA1.cpp

```diff
@@ -242,13 +242,10 @@ void SHA1::addUTF8Bytes(CFStringRef string)
 
 CString SHA1::hexDigest(const Digest& digest)
 {
-    char* start = nullptr;
+    std::span<char> start;
     CString result = CString::newUninitialized(40, start);
```

Comment:
I wonder why this uses the magic number 40. Seems safer to write `hashSize * 2`.